crates/fuel-core/src/schema/balance.rs

```diff
@@ -79,9 +79,11 @@ impl BalanceQuery {
         Ok(balance)
     }
 
-    // TODO: This API should be migrated to the indexer for better support and
```

Comment:
This is not something we want to do in the future anymore ? In my mind it's still something we want to do in the future to decouple both use cases of the node.

Reply:
Hard to tell, as far as I know, indexer is no longer a thing. But even if it is, we should track it via a separate issue, not a `TODO`.